tracing-appender/src/non_blocking.rs

```diff
@@ -277,7 +277,16 @@ impl Drop for WorkerGuard {
                 // when the `Worker` calls `recv()` on a zero-capacity channel. Use `send_timeout`
                 // so that drop is not blocked indefinitely.
                 // TODO: Make timeout configurable.
-                let _ = self.shutdown.send_timeout((), Duration::from_millis(1000));
+                match self.shutdown.send_timeout((), Duration::from_millis(1000)) {
+                    Err(SendTimeoutError::Disconnected(_)) => (),
+                    Err(SendTimeoutError::Timeout(e)) => {
+                        println!("Failed to wait for logging worker shutdown. Error: {:?}", e);
```

Comment:
The value inside of the `SendTimeoutError::Timeout` variant isn't actually an error value, it's the message that was sent (which in this case was `()`). So, this will just print "Error: ()", which doesn't actually seem very helpful.

What if instead, we dropped the `()` value and just printed that the send timed out, maybe like this:
```suggestion
                let timeout = Duration::from_millis(1000);
                match self.shutdown.send_timeout((), timeout) {
                    Err(SendTimeoutError::Disconnected(_)) => (),
                    Err(SendTimeoutError::Timeout(_)) => {
                        eprintln!("Shutting down logging worker timed out after {:?}.", timeout);
```

Reply:
(i also changed the error to be printed using `eprintln!` rather than `println!`, because it seemed like it belongs on `stderr` rather than `stdout`...)

Reply:
sure, this is better

added similar changes to the code slightly below fb1f941483cfab46cec8e5652957119e5c561cce
also fixed formatting